### PR TITLE
Swap Qunit for Tape, add better instructions to exercise1.js

### PR DESCRIPTION
**Pull request by AbdullahChaudhry**
#16

Swap Qunit tests for Tape
Added better instructions in exercise1.js
Moved global variables to the top inside exercise1.js

### CODE REVIEW

File: test/exercise1.test.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
-QUnit.test('Refactor our addOne function so it is pure.', function ( t ) {
+const tape = require('tape');
+const { addOne, timesTwo, incrementArray, addNumberArray, incrementObject, constantNumber, constantArray, constantObject } = require('../exercises/exercise1.js');
+
+
+tape('tape is working', function(t) {
```
Comment (Jwhiles) [inline test/exercise1.test.js]: you don't need this

File: exercises/exercise1.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
+// leave me alone :) :) :) <3 <3 <3
+var constantNumber = 5;
```
Comment (Jwhiles) [inline exercises/exercise1.js]: if we are using es6, we should change these all to consts

File: exercises/exercise1.js [hunk 3]
```diff
@@ -30,12 +41,13 @@ var incrementObject = function (object) {
...
-// leave me alone :) :) :) <3 <3 <3
-var constantNumber = 5;
-var constantArray = [5, 7, 23, 4];
-var constantObject = {
-  "a": 5,
-  "b": 2,
-  "c": 8
-};
-// leave me alone :) :) :) <3 <3 <3
+module.exports = {
```
Comment (Jwhiles) [inline exercises/exercise1.js]: you can just write
```js
module.exports = {
  addOne,
  incrementArray,
  addNumberArray


 etc etc
```

Comment (oliverjam) [reply exercises/exercise1.js]: The whole of Week 2 switched to Tape for FAC11, which means they will already have been introduced to Node and npm by this point. That's also why they want to change the workshop to use Tape—just for consistency with the rest of the week

**Comment by oliverjam** [comment]
@Jwhiles The whole of Week 2 switched to Tape for FAC11, which means they will already have been introduced to Node and npm by this point. That's also why they want to change the workshop to use Tape—just for consistency with the rest of the week

File: exercises/exercise1.js (hunk 2 above)
Comment (rachaelcodes) [reply exercises/exercise1.js]: my understanding is that ES6 is not introduced until a later week

**Comment by ZooeyMiller** [comment]
I accidentally merged this due to #2 including these changes, so sorry for not taking these into account. 
It seemed the only problem was from @Jwhiles about tape, but that was clarified, and the changes looked good to me otherwise.

If anyone wants me to roll this back I am happy to though.

**Pull request merged**